readme.md

```diff
@@ -120,6 +121,16 @@ require('d3-flamegraph')({
 })
 ```
 
+### `collapseHiddenNodeWidths`
+
+Boolean, affects the widths of stack frames excluded by type filters.
+
+This is an advanced option that is best used together with `heatBars: true` and a custom sort function that does something that is useful for your visualization.
+
+When false (default), hidden frames take up horizontal space, and visible children of hidden stack frames are aligned along where the hidden frame would be. In practice, this means there may be gaps between frames but toggling filters will only ever visually move frames vertically.
+
+When true, hidden frames do not take up space, but instead all their visible children are aligned closely next to each other, to the left of their closest visible parent. In practice, this means that there are no gaps between frames, instead the amount of free space on the right of a visible frame indicates the amount of time it was at the top of the stack. When toggling filters, frames may jump horizontally.
+
```

Comment:
I don't think this needs to be so long. Not wanting to leave gaps where something is hidden is a pretty straightforward preference - "why are there gaps" is a pretty common first time 0x question, for example.

I think just the final paragraph alone, starting "Boolean. When true, ...", is enough.

Reply:
Maybe also drop "the amount of free space on the right of a visible frame indicates the amount of time it was at the top of the stack" because that's only true if you have a certain type of non-default sort applied.

"In practice, this means there are no gaps between frames." seems pretty clear.